containers/EmpSponsors.ts

```diff
@@ -107,16 +127,64 @@ const useEmpSponsors = () => {
               isPricefeedInvertedFromTokenSymbol(tokenSymbol)
             );
 
-            newPositions[sponsor] = {
-              tokensOutstanding: position.tokensOutstanding,
-              collateral: position.collateral,
-              cRatio: cRatio.toString(),
-              liquidationPrice: liquidationPrice.toString(),
-              sponsor,
-            };
+            if (
+              position.tokensOutstanding !== "0" &&
+              position.collateral !== "0"
+            ) {
+              newPositions[sponsor] = {
+                tokensOutstanding: position.tokensOutstanding,
+                collateral: position.collateral,
+                cRatio: cRatio.toString(),
+                liquidationPrice: liquidationPrice.toString(),
+                sponsor,
+              };
+            }
+          });
+
+          empData.liquidations.forEach((liquidation: LiquidationQuery) => {
+            const sponsor = utils.getAddress(liquidation.sponsor.id);
+            const liquidationCreatedEvent = liquidation.events.find(
+              (e) => e.__typename === "LiquidationCreatedEvent"
+            );
+
+            // There should always be a LiquidationCreatedEvent associated with each liquidation object, but if not then
```

Comment:
if it is always the case then why include the if statement? how would there be a liquidation without the event?

Reply:
It'd be an indexing error but we should check just incase. If there for whatever reason is no `LiquidationCreatedEvent` then we can't trust any of the other data